templates/infra/deploy.bat

```diff
@@ -1,3 +1,4 @@
 cd infra
 npm install
+npx cdk bootstrap
```

Comment:
Looks good, I have one question. Which aws account/region will this bootstrap happen? I'm thinking it will happen on the default account on the user computer, but I'm not sure. I've taken a look at the docs and I think it will be necessary to use the stage info in this command.
`cdk bootstrap aws://ACCOUNT-NUMBER-1/REGION-1`

Reply:
You are correct.

When we run the `bootstrap` command, it uses these variables to deploy the CDK stack into the cloud formation in the region provided by the `AWS_DEFAULT_REGION` environment variable.

The problem is that the bootstrap command doesn't have a good way to override/extends the configurations as we do in /bin/infra.ts

I have only seen one way passing the `--profile` parameter, but it would take more configuration to create these profiles.

What do you think? Do you see any way to pass these variables dynamically to the bootstrap command?